real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,169 @@
+        string="Facades",
+        help="Number of facades of the property"
+    )
+    
+    garage = fields.Boolean(
+        string="Garage?",
+        help="Indicates whether the property has a garage"
+    )
+    
+    garden = fields.Boolean(
+        string="Garden?",
+        help="Indicates whether the property has a garden"
+    )
+    
+    garden_area = fields.Integer(
+        string="Garden Area",
+        help="The area of the garden in square meters"
+    )
+    
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', 'North'),
+            ('south', 'South'),
+            ('east', 'East'),
+            ('west', 'West')
+        ],
+        help="The orientation of the garden"
+    )
+    
+    active = fields.Boolean(
+        string="Active",
+        default=True,
+        help="Indicates whether the property is active and available for sale"
+    )
+    
+    state = fields.Selection(
+        string="Current State",
+        selection=[
+            ('new', 'New'),
+            ('offer_received', 'Offer Received'),
+            ('offer_accepted', 'Offer Accepted'),
+            ('sold', 'Sold'),
+            ('cancelled', 'Cancelled')
+        ],
+        required=True,
+        copy=False,
+        default='new',
+        help="The current state of the property"
+    )
+    
+    property_type_id = fields.Many2one(
+        "estate.property.type",
+        string="Property Type",
+        help="The type of the property (e.g., apartment, house, etc.)"
+    )
+    
+    buyer = fields.Many2one(
+        "res.partner",
+        string="Buyer",
+        help="The buyer who purchased the property"
+    )
+    
+    salesman = fields.Many2one(
+        "res.users",
+        string="Salesman",
+        default=lambda self: self.env.user,
+        help="The salesperson responsible for the property"
+    )
+    
+    tag_ids = fields.Many2many(
+        "estate.property.tags",
+        string="Tags",
+        required=True,
+        help="Tags associated with the property (e.g., luxury, beachfront)"
+    )
+    
+    offer_ids = fields.One2many(
+        "estate.property.offer",
+        "property_id",
+        help="Offers made for this property"
+    )
+    
+    total_area = fields.Float(
+        compute="_compute_total_area",
+        help="The total area of the property (living area + garden area)"
+    )
+    
+    best_price = fields.Float(
+        compute="_compute_best_price",
+        help="The best price offered for the property"
+    )
+    
+    @api.depends("living_area", "garden_area")
+    def _compute_total_area(self):
+        for record in self:
+            record.total_area = record.living_area + record.garden_area
+    
+    @api.depends("offer_ids.price")
+    def _compute_best_price(self):
+        for record in self:
+            if record.offer_ids:
+                record.best_price = max(record.offer_ids.mapped('price'))
```

Comment:
```suggestion
            if offers := record.offer_ids:
                record.best_price = max(offers.mapped('price'))
```
Did you know we can assign a value at the same time checking it in conditional statement?